src/components/Layout/Layout.scss

```diff
@@ -1,7 +1,8 @@
 @use "../../styles/variables" as *;
 
 .layout {
+
   height: 100vh;
-  width: 100vw;
+  width:79%;
```

Comment:
Use whole numbers (rounded to nearest 5) and use view width (vw) instead of percentages for reasons discussed.